git-send-email.perl

```diff
@@ -920,7 +935,7 @@ sub validate_address {
 			valid_re => qr/^(?:quit|q|drop|d|edit|e)/i,
 			default => 'q');
 		if (/^d/i) {
-			return undef;
+			return;
```

Comment:
IIRC, "return;" and "return undef;" are not semantically equivalent depending on the calling context (i.e. is the function called in scalar context or list context?), and PBP suggests that the former is less error prone most of the time.